### PR TITLE
docs(changelog) list changes for 35

### PR DESCRIPTION
**Pull request by h3xar0n**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Edited docs to ...]
* [Added cool new feature ...]
* ...

### Issues resolved

Fix #XXX

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [ ] [Adheres to Kong style guide](https://github.com/Kong/docs.konghq.com/blob/master/STYLEGUIDE.md)
- [ ] Spellchecked my updates
- [x] Tagged "Team Docs" as reviewers
- [ ] Ready to be merged

### CODE REVIEW

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -2,6 +2,142 @@
...
+  - Added methods to links in the sidebar on documentation in the default portal theme.
+  - Added hover states to drop-downs in the sidebar of documentation in the default portal theme.
+  - Added a right border to the sidebar in the default portal theme.
+  - Changed "Full Name" registration field to be a custom registration field, and can be managed through the custom fields -   interface in Kong Manager.
+  - Changed custom checkbox to native checkbox in the default developer portal theme on the dashboard.
+- **Core**
+  - New RBAC user tokens are not stored in plaintext. If upgrading to this version, any existing tokens will remain in plaintext until either a) the token is used in an Admin API call or b) the rbac_user record is PATCHed (even if the PATCH request includes the existing value of `user_token`).
+- **Plugins**
+  - **Response Transformer Advanced**: 
+    - Conditional transformations on response status through the new flag if_status, a subfield in all transformations (e.g., `add.if_status`, `remove.if_status`); the transformation only happens if the response code matches one of the codes in the `if_status field
```
Comment (gszr) [inline app/enterprise/changelog.md]: Missing closing backtick in if_status

Comment (gszr) [reply app/enterprise/changelog.md]: Could we somehow explicitly highlight which plugins are introduced in this release? Resp Transformer Advanced and Request Validator are two such examples.

Comment (h3xar0n) [reply app/enterprise/changelog.md]: Sure, maybe I'll put _new_ next to them? @levfishbluefish , thoughts?

Comment (h3xar0n) [reply app/enterprise/changelog.md]:
```suggestion
    - Conditional transformations on response status through the new flag if_status, a subfield in all transformations (e.g., `add.if_status`, `remove.if_status`); the transformation only happens if the response code matches one of the codes in the `if_status field`
```

Comment (h3xar0n) [reply app/enterprise/changelog.md]: **Response Transformer Advanced** _new_

File: app/enterprise/changelog.md [hunk 2]
```diff
@@ -2,6 +2,142 @@
...
+  - Added custom registration fields support and custom registration field interface.
+  - Added methods to links in the sidebar on documentation in the default portal theme.
+  - Added hover states to drop-downs in the sidebar of documentation in the default portal theme.
+  - Added a right border to the sidebar in the default portal theme.
+  - Changed "Full Name" registration field to be a custom registration field, and can be managed through the custom fields -   interface in Kong Manager.
+  - Changed custom checkbox to native checkbox in the default developer portal theme on the dashboard.
+- **Core**
+  - New RBAC user tokens are not stored in plaintext. If upgrading to this version, any existing tokens will remain in plaintext until either a) the token is used in an Admin API call or b) the rbac_user record is PATCHed (even if the PATCH request includes the existing value of `user_token`).
+- **Plugins**
+  - **Response Transformer Advanced**:
```
Comment (h3xar0n) [inline app/enterprise/changelog.md]:
```suggestion
  - **Response Transformer Advanced** _new_: 
```

Will look like this:
  - **Response Transformer Advanced** _new_

File: app/enterprise/changelog.md [hunk 3]
```diff
@@ -2,6 +2,142 @@
...
+  - Changed "Full Name" registration field to be a custom registration field, and can be managed through the custom fields -   interface in Kong Manager.
+  - Changed custom checkbox to native checkbox in the default developer portal theme on the dashboard.
+- **Core**
+  - New RBAC user tokens are not stored in plaintext. If upgrading to this version, any existing tokens will remain in plaintext until either a) the token is used in an Admin API call or b) the rbac_user record is PATCHed (even if the PATCH request includes the existing value of `user_token`).
+- **Plugins**
+  - **Response Transformer Advanced**: 
+    - Conditional transformations on response status through the new flag if_status, a subfield in all transformations (e.g., `add.if_status`, `remove.if_status`); the transformation only happens if the response code matches one of the codes in the `if_status field`
+    - Full-body replacement through the new config.replace.body
+    - Added a support of status code ranges for `if_status` configuration parameter. Now you can provide status code ranges and single status codes together (e.g., `201-204,401`)
+  - **Request Validator**:
```
Comment (h3xar0n) [inline app/enterprise/changelog.md]:
```suggestion
  - **Request Validator** _new_:
```

**Pull request merged**